### DISCUSSION TITLE
Resize to a specific length

### DISCUSSION THREAD

**Discussion by tbmuc** [ImageMagick/ImageMagick#3736] [Help] [answered]
Hello,

how can a image be resized to a specific length without changing the aspect ratio, Assume the target lenght is 1000

eg 300x400  (3:4) => 750x1000
eg 2000x3000 (2:3) = > 666:1000

**Comment by fmw42** [comment, marked as answer]
`convert input -resize x1000 output`

if on ImageMagick 7, change "convert" to "magick"
